storage/src/vespa/storage/storageserver/servicelayernode.cpp

```diff
@@ -39,6 +39,15 @@ ServiceLayerNode::ServiceLayerNode(const config::ConfigUri & configUri, ServiceL
 {
 }
 
+void
+ServiceLayerNode::report(vespalib::JsonStream &writer) const
+{
+    (void) writer;
```

Comment:
This seems unnecessary since the variable is referenced within the `if`

Reply:
Fixed